src/CoreShop/Component/Index/Getter/FieldCollectionGetter.php

```diff
@@ -45,7 +51,19 @@ public function get(IndexableInterface $object, IndexColumnInterface $config): m
 
         foreach ($validItems as $item) {
             if (method_exists($item, $fieldGetter)) {
-                $fieldValues[] = $item->$fieldGetter();
+                if (!empty((new \ReflectionMethod($item, $fieldGetter))->getParameters())) {
```

Comment:
feels a bit dirty.... we might wanna rather use ClassDefinition\FieldCollection Definition to check if the field is localized or not.

Reply:
Hope this one is more cleaner.